build.sbt

```diff
@@ -160,6 +160,7 @@ lazy val zioConfig = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(
     libraryDependencies ++= Seq(
       "dev.zio"                 %% "zio"                     % zioVersion,
+      "dev.zio"                 %% "zio-prelude"             % "1.0.0-RC6",
```

Comment:
I'm not sure if it should depend on `zio-prelude` for the `NEL` but alternatively we could still get the unsafe `head` from the the resultant list and replace `NEL` with a property check on `merge`.